examples/CalcQt/CMakeLists.txt

```diff
@@ -15,7 +15,9 @@ if(TARGET Qt::Core AND TARGET Qt::Gui AND TARGET Qt::Widgets AND TARGET Qt::Test
     add_executable(calcqt src/CalcQt.cpp)
     target_link_libraries(calcqt PRIVATE libcalcqt Qt::Widgets)
 
-    if(Qt::Test)
+    if(TARGET Qt::Test)
```

Comment:
Shouldn't we remove `AND TARGET Qt::Test` in the outer if statement?

Reply:
You're right.

Reply:
That is probably correct. They all depend on QTest, but only one uses it as a test runner.

Reply:
I wasn't sure if you resolved this 1 minute after my force push or it's a glitch so I unresolved this back again.

Reply:
I did resolve it tight after you pushed the commit, waiting for the build, as it did address my concerns :-)